### DISCUSSION TITLE
:set commands aren't working after latest update (1.9.1)

### DISCUSSION THREAD

**Discussion by dylanheiman** [JetBrains/ideavim#426] [Q&A] [answered]
I have a keybind in my .ideavimrc that toggles relative line numbers which comes in handy when pair programming. However, with the most recent Ideavim update my custom bind doesn't work. After poking around, I realized no `:set` commands are working. Has this problem happened to anyone else?

Here's the keybind in question: `nnoremap <leader>tl :set relativenumber!<cr>`

Any help would be greatly appreciated!

**Comment by AlexPl292** [comment, marked as answer]
Hi, sorry, that seems to be a bug in the new implementation of vim parser. We'll fix this and for the moment you can use the following rude mapping: `nnoremap <expr> <leader>tl  (&relativenumber) ? ':set norelativenumber<cr>' : ':set relativenumber<cr>'`

**Reply by lippfi** [reply]
This bug will be fixed in the next release, thank you for your report! 
Incidentally, since 1.9.0 IdeaVim supports `setlocal` command for `number` and `relativenumber` options and `setlocal` is not affected by the bug, so you can use it instead.
Have a nice day!

**Reply by dylanheiman** [reply]
Sounds good, thanks for your help!
